tutorials/introductory/usage.py

```diff
@@ -531,9 +560,9 @@ def my_plotter(ax, data1, data2, param_dict):
 #     plt.xlabel("index")
 #
 # and you will see the plot being updated after each line.  Since version 1.5,
-# modifying the plot by other means *should* also automatically
-# update the display on most backends. Get a reference to the :class:`~matplotlib.axes.Axes` instance,
-# and call a method of that instance::
+# modifying the plot by other means *should* also automatically update the
```

Comment:
I would reword this and the paragraph above to not mention mpl<1.5 (well that can be left to a note if you want to keep it, e.g. "note that if you are running mpl<1.5, etc.")